_playwright-tests/tests/API/SearchEnvFeature.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { test, expect, RepositoriesApi, EnvironmentsApi, ApiRepositoryResponse } from 'test-utils';
+import { poll } from 'test-utils/helpers';
+import { ListRepositoriesRequest } from 'test-utils/client';
+
+/**
+ * Test the API support for list / search environments using hardcoded EPEL repository
+ * This test validates that environment search endpoints work correctly for both URL and UUID searches.
+ */
+
+test.describe('Search Environment Endpoint', () => {
+  const testRepoUrl = 'https://dl.fedoraproject.org/pub/epel/10/Everything/x86_64/';
+  const keyword = 'kde';
+
+  test('Environment search with EPEL repository', async ({ client }) => {
+    const repositoriesApi = new RepositoriesApi(client);
+    const environmentsApi = new EnvironmentsApi(client);
+
+    let testRepo: ApiRepositoryResponse;
+
+    await test.step('Find existing EPEL repository', async () => {
+      const existingRepos = await repositoriesApi.listRepositories({
+        url: testRepoUrl,
+        origin: 'external',
```

Comment:
this should be `origin: 'community'` since the community_repos feature flag is on in CI